docs/user/attestations/publish/v1.md

```diff
@@ -0,0 +1,65 @@
+---
+title: PyPI Publish Attestation (v1)
+---
+
+<!--[[ preview('index-attestations') ]]-->
+
+Type URI: <https://docs.pypi.org/attestations/publish/v1>
+
+Version 1.0
+
+## Purpose
+
+To provide a minimal, "implicit" digital attestation for PyPI packages published
+via Trusted Publishing.
+
+## Use Cases
+
+A [Trusted Publisher] can produce this attestation during the publishing
+process for a particular release of a PyPI project. This allows consumers of
+that release version to verify the following:
+
+1. That a particular PyPI project release was, in fact, uploaded via a Trusted
+   Publisher and not some other publishing mechanism (such as a locally-held
+   API token).
+2. That a *specific* Trusted Publisher identity was used to publish the
+   release, such as a particular GitHub Actions workflow, GitLab identity,
+   etc.
+
+Put together, these allow users to assert a higher degree of confidence in
+the integrity (but not necessarily trustworthiness) of releases made to PyPI,
+by asserting that the package is published via a short-lived credential
+corresponding to a specific machine identity (such as a GitHub Actions workflow).
```

Comment:
This section implies attestations are for releases, not for individual files. The wording makes it seem that the uploading and verification happens at the release level, rather than at the file level (unless the term "PyPI release" can be used interchangeably for whole releases and individual files)

Reply:
Thanks, good catch! I'll clarify that the context is a *release distribution*, not the entire release.

Reply:
Updated to better emphasize that the project has the Trusted Publisher, and that individual project release distributions are what's signed. PTAL 🙂

Reply:
LGTM! I left a couple of nitpick comments